cpp/include/raft_runtime/neighbors/cagra_hnswlib.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <raft/core/host_mdspan.hpp>
+#include <raft/core/resources.hpp>
+#include <raft/neighbors/cagra_hnswlib_types.hpp>
```

Comment:
This is going to leak hnswlib types through the headers. Not good

Reply:
That file does not include any hnswlib headers.